EIPS/eip-4841.md

```diff
@@ -0,0 +1,425 @@
+---
+eip: 4841
+title: Expandable Onchain SVG Images Storage Structure
+description: It is a Expandable Onchain SVG Images Storage Structure Model on Ethereum.
+author: KyungEun Kim <kkekke815@gmail.com>, Soohan Park <725psh@gmail.com>, Soohan Park (@Soohan-Park)
+discussions-to: https://ethereum-magicians.org/t/eip-4841-expandable-onchain-svg-images-storage-structure/8410
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-02-23
+---
+
+
+
+## Simple Summary
+It is a Expandable Onchain SVG Images Storage Structure Model on Ethereum.
+
+
+
```

Comment:
```suggestion
```

Simple Summary isn't a valid EIP section.  See https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md for the correct layout.

Reply:
Confirmed :)